src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,12 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ moviesFromServer }) => (
```

Comment:
```suggestion
export const MovieList = ({ moviesFromServer = [] }) => (
```
Consider writing the default value to prevent further errors